var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -261,6 +261,9 @@ def nvariant_bool(feature):
                 cmake_args.extend([
                     '-DPARAVIEW_USE_QT:BOOL=%s' % variant_bool('+qt'),
                     '-DPARAVIEW_BUILD_WITH_EXTERNAL=ON'])
+                if spec.satisfies('%cce'):
+                    cmake_args.extend([
+                        '-DVTK_PYTHON_OPTIONAL_LINK:BOOL=OFF'])
```

Comment:
Since only adding a single option, why not use append?

Reply:
okay will adjust